pkg/csi/cephfsdeployment.go

```diff
@@ -178,13 +181,17 @@ var cephFSDeploymentSpec = appsv1.DeploymentSpec{
 
 func SetCephFSDeploymentDesiredState(deploy *appsv1.Deployment) {
 	// Copy required labels
-	for key := range cephfsDaemonsetLabels {
-		deploy.Labels[key] = cephfsDaemonsetLabels[key]
+	for key := range cephfsDeploymentLabels {
+		deploy.Labels[key] = cephfsDeploymentLabels[key]
 	}
 
 	// Update the deployment set with desired spec
 	cephFSDeploymentSpec.DeepCopyInto(&deploy.Spec)
 
+	podSpecTemplate := &deploy.Spec.Template
+	podSpecTemplate.Name = RBDDeploymentName
+	podSpecTemplate.Namespace = deploy.Namespace
```

Comment:
AFAIK This is not required and you are using `RBDDeploymentName` name here.

Reply:
> you are using RBDDeploymentName name here.
- good catch, thanks.

> This is not required
- we need to set pod name & ns isn't it?

Reply:
we need to set container names, pod names are auto generated from kubernetes based on the deployment name

Reply:
> pod names are auto generated
- I was just following previous behavior https://github.com/red-hat-storage/ocs-client-operator/blob/release-4.16/pkg/csi/cephfsdaemonset.go#L66-L68, do you want me to remove ns as well?

Reply:
done.